completion/ksbaduk.cwl

```diff
@@ -15,10 +15,10 @@ tmarkfont=%<font%>
 #endkeyvals
 
 ## 2.2 Global options
-\BadukpanSize{%<number%>}%definition
-\BadukpanColor{%<color%>%color}%definition
-\BackgroundColor{%<color%>%color}%definition
-\NumberFont{%<font%>%font}%definition
+\BadukpanSize{%<number%>}
+\BadukpanColor{%<color%>%color}
+\BackgroundColor{%<color%>%color}
```

Comment:
These should just be
```
\BadukpanColor{color}
\BackgroundColor{color}
```
though it would be nice to be able to specify color args this way

Reply:
Ah, sad